### ISSUE TITLE
Use Alexa rank for tracking-protection-issues-private bugs

### ISSUE THREAD

**Issue by adamopenweb** [webcompat/webcompat.com#2584]
For reports that come through the tracking-protection-issues-private repo, we'd like to be able to identify the Alexa global and local rank. 

A labeling system similar to the Webcompat `priority` labels might work.

**Comment by miketaylr** [comment]
I think it should be possible to decouple the priority labeling into its own webhook endpoint, then the TP private repo could send webhook data to that endpoint and get its own labels.

Depends on #2574 (kinda, not really...)

**Comment by karlcow** [comment]
@miketaylr @adamopenweb could you define a bit better what talks to what? And how this should work?

**Comment by miketaylr** [comment]
This is probably WONTFIX right now.
